__tests__/01_basic_spec.js

```diff
@@ -69,6 +66,7 @@ describe('basic spec', () => {
     const { getAllByText, container } = render(<App />);
     expect(container).toMatchSnapshot();
     fireEvent.click(getAllByText('+1')[0]);
```

Comment:
Now, we have only one button, can we do this?
```suggestion
    fireEvent.click(getByText('+1'));
```

Reply:
how about this : 
```js
- fireEvent.click(getAllByText('+1')[0]);
+ fireEvent.click(screen.getByRole('button'));
```
it's more clear and easy to maintain(now it's applied in my PR)

and about what said 

>  for testing more accurately from outside, we should check commit count.


I will be happy to work on it if you give me a hint.

Reply:
By commit count, I mean we count with `useEffect`. You don't need to change the current test, but if you are willing to add more tests, that would be a good way.